client/src/app/pages/applications/application-form/useApplicationForm.ts

```
@@ -0,0 +1,353 @@
+    sourcePlatformToRef,
+    sourcePlatformFromName,
+  },
+}: ApplicationFormProps) => {
+  const { t } = useTranslation();
+
+  const manualTagRefs: TagRef[] = useMemo(() => {
+    return application?.tags?.filter((t) => !t?.source) ?? [];
+  }, [application?.tags]);
+
+  const nonManualTagRefs = useMemo(() => {
+    return application?.tags?.filter((t) => t?.source ?? "" !== "") ?? [];
```

Biome (2.1.2)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)